scipy/interpolate/_rgi.py

```diff
@@ -171,17 +172,38 @@ class RegularGridInterpolator:
     ...                   alpha=0.4, label='ground truth')
     >>> plt.legend()
 
-    Other examples are given
+    **Length-one axes**
+
+    If one of the grid dimensions has length one, linear and nearest
+    interpolators will extrapolate in that direction, as  controlled by the
+    `fill_value`. For instance:
+
+    >>> x = np.array([0, 5., 10])
+    >>> y = np.array([0.])
+    >>> def f(x,y):
+    ...     return (y+1) * x
+    >>> data = f(*np.meshgrid(x,y, indexing='ij', sparse=True))
+    >>> interp = RegularGridInterpolator((x, y), data,
+    ...                                  fill_value=None, bounds_error=False)
+    >>> interp([[6, 0], [6, 1]])
+    array([6., 6.])
+
+    **Other examples** are given
     :ref:`in the tutorial <tutorial-interpolate_regular_grid_interpolator>`.
```

Comment:
I think we need a `plt.show()` somewhere above because this plot is showing up down here.
![image](https://user-images.githubusercontent.com/6570539/170180347-ce1ff3b5-0753-43f5-b04f-8ccebe438a4d.png)